### ISSUE TITLE
The results of Softmax are incorrect

### ISSUE THREAD

**Issue by daemyung** [kakaobrain/trident#13]
## 🐞 Describe the bug

If the batch size is bigger than 1 the results of Softmax are incorrect.

## 🧑‍🏫 Reproduction

Change `test_math.py` from:
```
cls.m = 1
```
To:
```
cls.m = random.randint(64, 16384)
```

## 🎯 Expected behavior

The results from Trident should be the same as PyTorch.
